crates/burn-cube/README.md

```diff
@@ -0,0 +1,213 @@
+<div align="center">
+<img src="../burn-cube/assets/logo.drawio.svg" width="400px"/>
+
+<br />
+<br />
+
+[![Rust Version](https://img.shields.io/badge/Rust-1.75.0+-blue)](https://releases.rs/docs/1.75.0)
+![license](https://shields.io/badge/license-MIT%2FApache--2.0-blue)
+
+---
+
+**Multi-platform high-performance compute language for Rust.**
+<br/>
+
+</div>
+
+## TL;DR
+
+With CubeCL, you can use Rust to program your GPU, any GPU!
+
+## Motivation
+
+The goal of CubeCL is to ease the pain of writing highly optimized compute kernels that are portable across hardware.
+There is currently no adequate solution when you want optimal performance while still being multi-platform.
+You either have to write custom kernels for different hardware, often with different languages such as CUDA, Metal, or ROCm.
+To make it possible, we created a Just-in-Time compiler with three core features: **automatic vectorization**, **comptime**, and **autotune**!
+
+These features are extremely useful for anyone writing high-performance kernels, even when portability is not a concern.
+They improve code composability, reusability, and maintainability, all while staying optimal.
+
+### Disclaimer & History
+
+CubeCL is currently in **alpha**.
+The only supported runtimes are CUDA and WebGPU for now.
+It's easy to add more GPU runtimes and we intend to support Metal, ROCm, and Vulkan; contributions are welcome!
+We also want to have an optimized JIT CPU runtime with SIMD instructions, leveraging [Cranelift](https://cranelift.dev).
+
+While CubeCL is currently in use in [Burn](https://burn.dev), the user experience, such as error messages and other edge cases, isn't refined yet.
+The project started as a WebGPU-only backend for Burn.
+As we optimized it, we realized that we needed an intermediate representation (IR) that could be optimized then compiled to WGSL.
+Having an IR made it easy to support another compilation target, so we made a CUDA runtime.
+However, writing kernels directly in that IR wasn't easy, so we created a Rust frontend using the [syn](https://github.com/dtolnay/syn) crate.
+Navigating the differences between CUDA and WebGPU, while leveraging both platforms, forced us to come up with general concepts that worked everywhere.
+Hence, CubeCL was born!
+
+## Design
+
+CubeCL is designed around - you guessed it - Cubes! More precisely, cuboids since not all axes are forced to be the same size.
```

Comment:
More specifically, it's based on cuboids, because not all axes are the same size.

Using because will help differentiating from next paragraph starting with since